crates/x11bridge/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "muvm-x11bridge"
```

Comment:
Shall we rename the directory to `muvm-x11bridge` too to match the package name?

Reply:
Done!